app/views/maintenance.scala.html

```diff
@@ -0,0 +1,19 @@
+@import models.user.User
+@(title: String, user: Option[User] = None)(implicit lang: Lang)
+
+@main(title) {
```

Comment:
Is there a reason you're including main? It's probably best to just exclude this entirely. If part of our site isn't working, we want to try and use as little external code as possible on this page.

Reply:
And note that I didn't notice that this was here until I read the code or tested it. The screenshot you added to the PR didn't show the entire page, it was missing the footers. When you add screenshots showing what something looks like, make sure that you show as much of it as you possibly can through screenshots / videos.

Reply:
On a related note, the screenshots should have been posted in the Github issue first instead of creating a PR. Then I could give you feedback about the design (like removing the footer). Then you would create the PR.